src/libraries/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -2075,16 +2076,7 @@ public bool TryGetValue(T equalValue, [MaybeNullWhen(false)] out T actualValue)
         }
 
         // Used for set checking operations (using enumerables) that rely on counting
-        private static int Log2(int value)
-        {
-            int result = 0;
-            while (value > 0)
-            {
-                result++;
-                value >>= 1;
-            }
-            return result;
-        }
+        private static int Log2(int value) => BitOperations.Log2((uint) value);
```

Comment:
Isn't better to just remove this method and use `BitOperations.Log2` everywhere?

Reply:
It might be; I will check later whether it can be done without introducing too many parentheses due to casting to `uint`.

Reply:
I think the helper method approach is preferable to introducing casts everywhere.